testplan/common/entity/base.py

```diff
@@ -666,6 +666,7 @@ class Runnable(Entity):
     STATUS = RunnableStatus
     RESULT = RunnableResult
     ENVIRONMENT = Environment
+    _STOP_TIMEOUT = 5
```

Comment:
This timeout here is quite arbitary. It might be easier to just set _ihandler thread as daemon.

Reply:
Yes I will remove it, I found there is an cfg option `abort_wait_timeout` available for this. Also I set `thread.daemon` to True for safety reason. But I still keep `interruptible_join` there to allow interactive handler quit gracefully.